AppServer_Java/src/com/google/appengine/tools/resources/ResourceLoader.java

```diff
@@ -7,21 +7,16 @@
     private static ResourceLoader res                       = null;
     private final int             PROTOCOL_BUFFER_PORT      = 8888;
     private final String          DB_LOCATION_PROPERTY      = "DB_LOCATION";
-    private static String         apphome                   = "/root/appscale";
```

Comment:
hadoop is still needed as a dependency. I think we should remove it on a different PR.